### PR TITLE
update readme and config.js

### PR DESCRIPTION
**Pull request by schurigh**
more detailed and updated readme and prepared config.js for easy switch to local development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,16 @@
 
 ## Setup
 ### Installation
+More detailed (german) install instructions here: [Kalender Setup](https://docs.schul-cloud.org/display/SCDOK/Setup#Setup-Kalender)
 1. Install & start PostgreSQL
-2. Create a database named `schulcloud_calendar` and a user `node` (without password, but with permissions for the db)
-3. Clone project & run `npm install`
-4. Run `npm start`, the app should now be running on the specified port (defaults to 3000)
+2. Create a database named `schulcloud_calendar` and a user `node`, with password `node` and with `all` permissions for the db
```
Comment (Langleu) [inline README.md]: you can create users without password.
sometimes the `man` command helps.
psql has a --no-password option

File: README.md [hunk 2]
```diff
@@ -12,12 +12,16 @@
...
+3. Import DB data (view below)
+4. Clone project & run `npm install`
+5. Edit src/config.js and switch comments for local development
+6. Run `npm start`, the app should now be running on the specified port (defaults to 3000)
 
 ### Example data
+psql.exe must be accessable through shell/cmd (environment variables) or provide full path (in PostgreSQL/bin folder)
```
Comment (Langleu) [inline README.md]: not everyone is using windows...
On mac/linux psql is automatically in the path, windows might be not the best choice for web programming 😁

File: README.md [hunk 3]
```diff
@@ -12,12 +12,16 @@
 
 ## Setup
 ### Installation
+More detailed (german) install instructions here: [Kalender Setup](https://docs.schul-cloud.org/display/SCDOK/Setup#Setup-Kalender)
 1. Install & start PostgreSQL
-2. Create a database named `schulcloud_calendar` and a user `node` (without password, but with permissions for the db)
-3. Clone project & run `npm install`
-4. Run `npm start`, the app should now be running on the specified port (defaults to 3000)
+2. Create a database named `schulcloud_calendar` and a user `node`, with password `node` and with `all` permissions for the db
+3. Import DB data (view below)
```
Comment (Langleu) [inline README.md]: so you first import the data from a script that you only clone in the next step?
Teach me more magic, master 🧙‍♂️

File: README.md (hunk 1 above)
Comment (Langleu) [reply README.md]: btw. it's funny that you provided a password here, but later when importing you don't mention anything about it anymore

File: src/config.js [hunk 4]
```diff
@@ -5,11 +5,20 @@
 /**
  * Database settings for production environment
  */
+
+// START switch comment blocks to switch local development
 exports.DB_HOST = process.env.DB_HOST;
-exports.DB_PORT = process.env.DB_PORT || 5432;
+exports.DB_DATABASE = process.env.DB_DATABASE;
```
Comment (Langleu) [inline src/config.js]: what exactly is happening here?
Even if you do it like this, why not just set a default?
`exports.DB_DATABASE = process.env.DB_DATABASE || 'schulcloud_calendar'`

File: src/config.js [hunk 5]
```diff
@@ -21,10 +30,16 @@ exports.DOMAIN_NAME = process.env.DOMAIN_NAME || 'schul-cloud.org';
 exports.ROOT_URL = process.env.ROOT_URL || `https://${exports.DOMAIN_NAME}:3000`;
 
 /**
- * API Paths
- */
+* API Paths
+*/
+// START switch comment blocks to switch local development
```
Comment (Langleu) [inline src/config.js]: I think all these development things belong in a readme and not written as a comment in the actual file. Makes it look kinda ugly.
Or provide a script in the package.json with all the env variables.

File: README.md (hunk 2 above)
Comment (schurigh) [reply README.md]: although mac/linux users can ignore it, the info isnt wrong :P see it as a lame-windows-hint ;)

File: README.md (hunk 1 above)
Comment (schurigh) [reply README.md]: node user without password didn't work for me locally, calendar throws errors on start. so 
passwort was needed.

File: README.md (hunk 2 above)
Comment (Langleu) [reply README.md]: I just don't like the .exe part ^^

File: package.json [hunk 6]
```diff
@@ -4,6 +4,8 @@
   "private": true,
   "scripts": {
     "start": "node ./src/infrastructure/server",
+    "start_local": "DB_HOST=localhost DB_USERNAME=node DB_DATABASE=schulcloud_calendar SCHULCLOUD_ALL_USERS_FOR_UUID=http://localhost:3030/resolve/users/ SCHULCLOUD_ALL_SCOPES_FOR_TOKEN=http://localhost:3030/resolve/scopes/&& node ./src/infrastructure/server",
```
Comment (Langleu) [inline package.json]: && am Ende weg und am besten nodemon nehmen, dann startet der Server automatisch neu bei file changes

**Pull request merged**